src/test/components/CallTreeSidebar.test.js

```diff
@@ -181,12 +181,12 @@ describe('CallTreeSidebar', function () {
       funcNamesDict: { A, B, C },
     } = setup(getProfileWithSubCategories());
     selectNode([A, B, C]);
-    expect(queryByText('FakeSubCategoryC')).not.toBeInTheDocument();
+    expect(getByText('FakeSubCategoryC')).toBeInTheDocument();
 
     const layoutCategory = getByText('Layout');
     fireFullClick(layoutCategory);
 
-    expect(getByText('FakeSubCategoryC')).toBeInTheDocument();
+    expect(queryByText('FakeSubCategoryC')).not.toBeInTheDocument();
 
     expect(container.firstChild).toMatchSnapshot();
```

Comment:
Also this snapshot doesn't test the expanded state anymore (see the removed snapshot lines in the snap file). It would be good to add another `expect(container.firstChild).toMatchSnapshot();` line before we fire a full click so we can also see the expanded state in the snapshots.

Reply:
That's a good idea.